Resources/translations/fr/content_type.fr.xlf

```diff
@@ -0,0 +1,147 @@
+        <target>Le groupe de type de contenu "%identifier%" a été correctement créé.</target>
+      </trans-unit>
+      <trans-unit id="c20c719ac1ac6b400867977af4a0c6e1">
+        <source>content_type.group.notification.updated</source>
+        <target>Le groupe de type de contenu "%identifier%" a été correctement mis à jour.</target>
+      </trans-unit>
+      <trans-unit id="20d385d52df00a6a1aa4d01c7e75f4e5">
+        <source>content_type.group.deleted</source>
+        <target>Le groupe de type de contenu "%identifier%" a été correctement supprimé.</target>
+      </trans-unit>
+      <trans-unit id="147661b50276b00ffac1bb715bdb4746">
+        <source>content_type.group.cannot_delete.has_content_types</source>
+        <target>Impossible de supprimer le groupe de contenu "%identifier%" car il contient toujours des types de contenu.</target>
+      </trans-unit>
+      <trans-unit id="f04e48d37e0034d0db7c2fe532d0a98f" resname="content_type.name">
+        <source>content_type.name</source>
+        <target>Nom</target>
+      </trans-unit>
+      <trans-unit id="3654d0b83af5b21165d08ed00c583b24" resname="content_type.id">
+        <source>content_type.id</source>
+        <target>ID</target>
+      </trans-unit>
+      <trans-unit id="b7c7cef3a977522a9eec90f49adfc04e" resname="content_type.identifier">
+        <source>content_type.identifier</source>
+        <target>Identifiant</target>
+      </trans-unit>
+      <trans-unit id="c3f683a33cfccf890c919f6432af7877" resname="content_type.edit">
+        <source>content_type.edit</source>
+        <target>Éditer</target>
+      </trans-unit>
+      <trans-unit id="972127730f863287531a351da529a802" resname="content_type.edit_title">
+        <source>content_type.edit_title</source>
+        <target>En édition</target>
+      </trans-unit>
+      <trans-unit id="bd31df6edfdfe616bf43312ee6ed4814" resname="content_type.content_count">
+        <source>content_type.content_count</source>
+        <target>{0} %count% éléments de contenu|{1} %count% élément de contenu|]1,Inf[ %count% éléments de contenu</target>
+      </trans-unit>
+      <trans-unit id="978d4d6fcaabe95baf297bcddced946c" resname="content_type.modified_date">
+        <source>content_type.modified_date</source>
+        <target>Date de modification</target>
+      </trans-unit>
+      <trans-unit id="39f42e02559368a6b6ea2bb9f6de6724" resname="content_type.last_modified">
+        <source>content_type.last_modified</source>
+        <target>Dernière modification: %date% par %modifier%</target>
+      </trans-unit>
+      <trans-unit id="7c0ab0f708330c0d934c7e53a0cff93e" resname="content_type.description">
+        <source>content_type.description</source>
+        <target>Description</target>
+      </trans-unit>
+      <trans-unit id="e1d223825a9bf3ad0d1f810b7426bf3d" resname="content_type.name_schema">
+        <source>content_type.name_schema</source>
+        <target>Content name schema</target>
```

Comment:
that's a tricky one :) in legacy we translated more less the same expression in: *Modèle de nom d'objet* so I guess we can write something like: *Modèle de nom des contenus*